addon/templates/components/ember-sparkles/axis.hbs

```diff
@@ -0,0 +1,71 @@
+{{#with (hash
+  position=position
+  ticks=ticks
```

Comment:
Since you're not applying any transformations to this value, you can just use the ticks property out of the scope. ie. `ticks` instead of `options.ticks` further on.